Tasks/NodeToolV0/make.json

```diff
@@ -1,4 +1,11 @@
 {
+  "common": [
```

Comment:
This should be removed otherwise build script adds this dependency to package.json.